omg/entities/elements.py

```diff
@@ -0,0 +1,35 @@
+import os
+import json
+from typing import Dict
+
+ELEMENTS_JSON_DIR = os.path.join(os.path.dirname(__file__), "Elements.JSON")
+
+
+class DotDict(dict):
+    """Convert dictionary to a format to call attributes with dot notation."""
+
+    def __getattr__(self, attr):
+        """Get dictionary item as attribute.Convert nested dictionaries to DotDict."""
+        value = self.get(attr)
+        if isinstance(value, dict):
+            value = DotDict(value)
```

Comment:
this if logic will be true, right? As `DotDict` inherits from `dict`, and all your values are `DotDict`s, so `isinstance(value,dict)` is also true for `DotDict`.

Ideally what I imagine is that you would create this DotDict instances using `to_dotdict()` method you created. That method already created DotDict with each value being a DotDict. So, why would you need another `value = DotDict(value)?`. 

Even if you need it for some reason, I would expect that to be done in the `setattr` when you are actually adding a value to your dictionary

Reply:
Hey Alp,

- Yes, you are right. This `if isinstance(value, dict):` is true only if value is a dictionary or DotDict. 

- Yes, you are right. I forgot to remove it from there. it will be done in the next push